warp/payload/block_hash_payload.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package payload
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// BlockHashPayload includes the block hash
+type BlockHashPayload struct {
+	BlockHash common.Hash `serialize:"true"`
```

Comment:
Would this message be more useful to applications if it included the block number as well?

Reply:
I considered the DevX tradeoff of including that information here and decided against including it since a user should be able to look up the height for a corresponding block height.